src/main/java/org/mockito/runners/MockitoJUnitRunner.java

```diff
@@ -27,7 +29,7 @@
     @Deprecated
     public static class Silent extends MockitoJUnitRunner {
```

Comment:
The code works as designed (https://github.com/mockito/mockito/issues/1212#issuecomment-340266646).

If you want to keep the PR I suggest that we add a unit test that uses deprecated runner so that we preserve and document this behavior.